README.md

```diff
@@ -2,25 +2,39 @@
 
 [![GitHub license](https://img.shields.io/github/license/PX4/px4_msgs.svg)](https://github.com/PX4/px4_msg/blob/master/LICENSE) [![Build package](https://github.com/PX4/px4_msgs/workflows/Build%20package/badge.svg)](https://github.com/PX4/px4_msgs/actions)
 
-This package contains the ROS2 message definitions of the [PX4 Pro ecosystem](https://px4.io/). Building this package generates all the required interfaces to interface ROS2 nodes with the PX4 autopilot internals, which use the [uORB messaging API](https://dev.px4.io/en/middleware/uorb.html). Currently the messages of this package represent a dependency to [`px4_ros_com` package](https://github.com/PX4/px4_ros_com).
+This interface package contains the ROS 2 message definitions of the [PX4 Pro ecosystem](https://px4.io/). Building this package generates all the required interfaces to interface ROS 2 nodes with the PX4 autopilot internals.
 
-## uORB message definitions
+## Supported versions and compatibility
 
-The uORB message definitions, which represent the counter-part of the ROS2 messages found in this package, can be found on the [PX4 Firmware repository](https://github.com/PX4/Firmware).
+Depending on the PX4 and ROS versions you want to use, you need to checkout the appropriate branch of this package:
 
-## How are these messsage definitions generated?
+| PX4 version   | ROS 2 versions          | branch                                                            |
+|---------------|-------------------------|-------------------------------------------------------------------|
+| 1.13          | Foxy                    | [release/1.13](https://github.com/PX4/px4_msgs/tree/release/1.13) |
+| 1.14          | Foxy - Humble - Rolling | [release/1.14](https://github.com/PX4/px4_msgs/tree/release/1.14) |
+| `main` branch | Foxy - Humble - Rolling | [main](https://github.com/PX4/px4_msgs)                           |
 
-When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository change, a CI/CD pipeline automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
+### Manual messages synchronization
+
+When PX4 message definitions in the `main` branch of [PX4 Autopilot](https://github.com/PX4/Autopilot) change, a CI/CD pipeline automatically copies and pushes updated ROS message definitions to this repository. This ensures that this repository `main` branch and the PX4-Autopilot `main` branch are always up to date.
```

Comment:
Consider adding a link to the CI workflow and actions.

* [Workflow](https://github.com/PX4/PX4-Autopilot/blob/main/.github/workflows/metadata.yml#L119) 
* [Actions](https://github.com/PX4/PX4-Autopilot/actions/workflows/metadata.yml)

Reply:
Yes! I added the Workflow link.